### PR TITLE
feat(posts): add post about hive talks

### PR DESCRIPTION
**Pull request by dariagrudzien**
Could I please ask for your review on both the content and style of the post?

### CODE REVIEW

**Comment by char0n** [comment]
@dariagrudzien w8 until I merge [this](https://github.com/apiaryio/apiaryio.github.com/pull/114). I've fixed the tests not passing in that PR.

**Comment by char0n** [comment]
Ok go ahead, after merging to master the tests still did not pass but it successfully deployed.

File: _posts/2018-02-19-Hive-Talks-Security.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+Those three fields will come together on the computer security focused meetup oganized as part of our Hive Talk series on 6th of March, 2018.
+
+![Github Repos Association](/images/2018-02-19-Hive-Talks-Security/2018-02-19-Hive-Talks-Logo.png)
+
+Join us, grab a drink and enjoy the talks:
+
+- Petr Svenda ([Enigma Bridge](https://enigmabridge.com/), [Centre for Research on Cryptography and Security](https://crocs.fi.muni.cz/) at Masaryk University) is one of the Czech researchers credited with discovering [vulnerabilities in chips used in Estonias IDs - among others](https://arstechnica.com/information-technology/2017/10/crypto-failure-cripples-millions-of-high-security-keys-750k-estonian-ids/). Petr will talk about the troubles with RSA cryptographic keypair generation.
+
+- Dite Gashi ([Cryptelo](https://www.cryptelo.com/), [Decissio](https://decissio.com/), [Bethereum](https://www.bethereum.com/)) will talk about blockchain powered trust-less data storing and processing. One of Dite's projects, Cryptelo, raised $1 Million USD in a single night of its ICO pre-sale.
```
Comment (kylef) [inline _posts/2018-02-19-Hive-Talks-Security.md]: Should `storing` be `storage`?

I think `blockchain powered` should become `blockchain-powered`.

---

> One of Dite's projects, Cryptelo, raised $1 Million USD in a single night of its ICO pre-sale.

I think this should be `during` not `in` to make the grammar of this sentence work.

```diff
- Million USD in a single night of its ICO pre-sale
+ Million USD during a single night of its ICO pre-sale
```

File: _posts/2018-02-19-Hive-Talks-Security.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+excerpt: Join the meetup we're organizing at Apiary's Prague office on 6th of March
+layout: post
+date: 2018-02-19 10:00:00 +0200
+author: daria
+published: true
+---
+
+## Cryptography, Blockchain and Machine Learning
+
+Those three fields will come together on the computer security focused meetup oganized as part of our Hive Talk series on 6th of March, 2018.
```
[the remark this answers is not in the document]
Comment (w-vi) [reply _posts/2018-02-19-Hive-Talks-Security.md]: TYPO: Hive Talks

**Comment by dariagrudzien** [comment]
Thank you, corrected the typos and added a link to maps.

File: _config.yml [hunk 3]
```diff
@@ -109,6 +109,11 @@ authors:
     display_name: Honza Javorek
     email: "honza@apiary.io"
     twitter: "@honzajavorek"
+  daria:git log
```
Comment (kylef) [inline _config.yml]: Oh, `git log` is here. I think that was by accident. @dariagrudzien can you please remove.

**Comment by kuba-kubula** [comment]
btw - the Meetup is already full for some days, so I really feel like we're doing nice marketing, but a bad marketing at the same time.

Next time we will do better 👍

**Pull request merged**